apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -165,6 +166,13 @@ const ShareDialog = React.createClass({
       modalClass += ' no-modal-icon';
     }
 
+    var thumbnailImageUrl;
+    if (this.props.thumbnailUrl) {
+      thumbnailImageUrl = this.props.thumbnailUrl;
+    } else {
+      thumbnailImageUrl = "https://studio.code.org/blockly/media/projects/project_default.png";
+    }
```

Comment:
Our style guidelines have changed since this file was written. Please write this as:
```
const thumbnailImageUrl = this.props.thumbnailUrl ?
  this.props.thumbnailUrl :
  "/blockly/media/projects/project_default.png";
```
Three things to note:
1. prefer `const` (or `let`) over `var`
2. prefer the `?:` ternary operator for conditional assignments
3. use the relative path `/blockly/media/projects/project_default.png` so that we don't get any cross-domain problems when running this in other environments besides production